code/modules/uplink/uplink_item.dm

```diff
@@ -518,7 +518,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 /datum/uplink_item/stealthy_weapons/martialarts
 	name = "Martial Arts Scroll"
 	desc = "This scroll contains the secrets of an ancient martial arts technique. You will master unarmed combat, \
-			deflecting all ranged weapon fire, but you also refuse to use dishonorable ranged weaponry."
+			deflecting all ranged weapon fire and learning several fighting techniques, but you also refuse to use dishonorable ranged weaponry."
```

Comment:
This addition is already implied by the first sentence and first part of the second sentence, as well as by the name of the item.